homeassistant/components/notify/nfandroidtv.py

```diff
@@ -24,20 +24,24 @@
 CONF_TRANSPARENCY = 'transparency'
 CONF_COLOR = 'color'
 CONF_INTERRUPT = 'interrupt'
+CONF_ICON = 'icon'
 
 DEFAULT_DURATION = 5
 DEFAULT_POSITION = 'bottom-right'
 DEFAULT_TRANSPARENCY = 'default'
 DEFAULT_COLOR = 'grey'
 DEFAULT_INTERRUPT = False
 DEFAULT_TIMEOUT = 5
+DEFAULT_ICON = None
```

Comment:
This isn't used.

Reply:
Thanks for catching this. Now it is being used. :)